ignite/cmd/network_chain_init.go

```diff
@@ -76,21 +77,17 @@ func networkChainInitHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	if !getYes(cmd) && exist {
-		prompt := promptui.Prompt{
-			Label: fmt.Sprintf("The chain has already been initialized under: %s. Would you like to overwrite the home directory",
-				chainHome,
-			),
-			IsConfirm: true,
-		}
-		nb.Spinner.Stop()
-		if _, err := prompt.Run(); err != nil {
+		question := fmt.Sprintf(
+			"The chain has already been initialized under: %s. Would you like to overwrite the home directory",
+			chainHome,
+		)
+		if err := session.AskConfirm(question); err != nil {
 			fmt.Println("said no")
 			return nil
 		}
-		nb.Spinner.Start()
 	}
 
-	n, err := nb.Network()
+	n, err := nb.Network(network.CollectEvents(session.EventBus()))
```

Comment:
Already passed during newNetworkBuilder. We should somehow prevent passing it to Network and Chain by mistake.